### PR TITLE
Optionally allow to use some improvement to fuzzaldrin.

### PR DESCRIPTION
**Pull request by jeancroy**
As per this discussion first step is testing new scoring algorithm as an option (disabled by default)
https://github.com/atom/fuzzaldrin/pull/22#issuecomment-142701468

Screen

### CODE REVIEW

File: lib/fuzzy-finder-view.coffee [hunk 1]
```diff
@@ -1,20 +1,24 @@
 path = require 'path'
-{Point} = require 'atom'
-{$$, SelectListView} = require 'atom-space-pen-views'
+{Point, CompositeDisposable} = require 'atom'
+{$, $$, SelectListView} = require 'atom-space-pen-views'
 {repositoryForPath} = require './helpers'
 fs = require 'fs-plus'
-{match} = require 'fuzzaldrin'
+fuzzaldrin = require 'fuzzaldrin'
+fuzzaldrin_plus = require 'fuzzaldrin-plus'
```
Comment (benogle) [inline lib/fuzzy-finder-view.coffee]: Super tiny, but can you change this to `fuzzaldrinPlus`?

**Comment by benogle** [comment]
I've been messing around with this, and it seems like an improvement. Other than the one tiny change, I'm down to merge this.

**Comment by jeancroy** [comment]
I had a lot of example where current fuzzaldrin was not great, but people don't complain when something works. So maybe we have reversed some of the things that worked well. If it's the case I'll have no problem doing additional tuning.

**Comment by benogle** [comment]
Sounds good :+1: I'll point them to the fuzzaldrin-plus repo for any issues

**Comment by benogle** [comment]
This is now in atom/master

**Comment by Soleone** [comment]
:heart: thanks a lot guys

**Comment by JesseLeite** [comment]
<3

**Comment by benogle** [comment]
Forgot to say this but: thanks @jeancroy for working on this. You obviously put a ton of work into it. I especially like the discussion in the readme/body of the issue.

**Comment by walles** [comment]
Available in fuzzy-finder 0.90.0.

Thanks @jeancroy for implementing and driving this!

**Comment by mjohnsonengr** [comment]
Took me a bit to realize this option existed in Fuzzy Finder (or that Fuzzy Finder was the source of my frustrations), but once I found it, This is REALLY FREAKING AWESOME! Thanks! :grin: :fireworks:

**Pull request merged**